samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,114 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+DT_CHOSEN_EXT_FLASH:= nordic,pm-ext-flash
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	imply MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	imply MCUMGR_CMD_IMG_MGMT
+	imply MCUMGR_CMD_OS_MGMT
+	imply MCUMGR_SMP_REASSEMBLY_BT
+	imply NORDIC_QSPI_NOR if $(dt_chosen_enabled,$(DT_CHOSEN_EXT_FLASH))
+	imply NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	depends on BT_PERIPHERAL
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+if NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+
+config MCUMGR_SMP_BT_AUTHEN
+	default n
+
+config IMG_ERASE_PROGRESSIVELY
+	default y
+
+if MCUMGR_SMP_REASSEMBLY_BT
+
+config MCUMGR_BUF_SIZE
+	default 2475
+
+config OS_MGMT_MCUMGR_PARAMS
+	default y
+
+endif # MCUMGR_SMP_REASSEMBLY_BT
+
+if (SETTINGS_FCB || SETTINGS_NVS)
+
+# Enable support for the storage erase operation only when the storage
+# partition is created by the Partition Manager.
+
+config MCUMGR_GRP_ZEPHYR_BASIC
+	default y
+
+config MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	default y
+
+endif # (SETTINGS_FCB || SETTINGS_NVS)
+
+if NORDIC_QSPI_NOR
+
+config NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE
+	default 4096
+
+config NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE
+	default 16
+
+endif # NORDIC_QSPI_NOR
+
+# Align supported targets with their default MCUboot configuration.
+
+# Thingy53: mcuboot/boot/zephyr/boards/thingy53_nrf5340_cpuapp.conf
+if BOARD_THINGY53_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP_NS
+
+config UPDATEABLE_IMAGE_NUMBER
+	default 2
+
+endif # BOARD_THINGY53_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP_NS
+
+# nRF5340 DK: mcuboot/boot/zephyr/prj.conf
+if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+
+config NRF53_UPGRADE_NETWORK_CORE
+	default n
+
+endif # BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS
```

Comment:
Since we use common MCUboot configuration here, maybe we could rely on the fact that we use nRF53 SoC and we are not thingy:53 (then we could also handle properly other nRF53 based boards)

Reply:
Addressed by my latest force push:

https://github.com/nrfconnect/sdk-nrf/compare/6ce9a447f66309e162175a7190a23f0a6b5f94cc..22658370598965443b2d241c0712ad7d14914ee2